third_party/2and3/click/testing.pyi

```diff
@@ -0,0 +1,74 @@
+from typing import (IO, Any, BinaryIO, ContextManager, Dict, Iterable, List,
+                    Mapping, Optional, Tuple, Union)
+
+from .core import BaseCommand
+
+clickpkg: Any
+
+class EchoingStdin:
+    def __init__(self, input: BinaryIO, output: BinaryIO) -> None: ...
+    def __getattr__(self, x: str) -> Any: ...
+    def read(self, n: int = ...) -> bytes: ...
+    def readline(self, n: int = ...) -> bytes: ...
+    def readlines(self) -> List[bytes]: ...
+    def __iter__(self) -> Iterable[int]: ...
+
+def make_input_stream(input: IO, charset: str) -> BinaryIO: ...
+
+class Result:
+    runner: CliRunner = ...
+    stdout_bytes: bytes = ...
+    stderr_bytes: Any = ...
```

Comment:
This doesn't exist on the version of click I just installed, or in the documentation (http://click.pocoo.org/5/api/#click.testing.Result).

Same for the stdout and stderr properties.

Reply:
Never mind, apparently they exist on master: http://click.pocoo.org/dev/api/#click.testing.Result.